sycl/source/virtual_mem.cpp

```diff
@@ -58,6 +58,9 @@ __SYCL_EXPORT size_t get_mem_granularity(const device &SyclDevice,
   Plugin->call(urVirtualMemGranularityGetInfo, ContextImpl->getHandleRef(),
                DeviceImpl->getHandleRef(), GranularityQuery, sizeof(size_t),
                &Granularity, nullptr);
+  assert(
```

Comment:
I'm not sure this will help with Coverity. `assert` only runs in debug mode, so the problem still exists in release mode.

Reply:
My bad, I didn't know that. I am changing the assert to a throw instead.